public_src/assets/i18n/cs.json

```diff
@@ -59,12 +61,20 @@
   "Highlight Modes": "Highlight Modes",
   "Incomplete relation (click to download it by reference ID)...": "Incomplete relation (click to download it by reference ID)...",
   "Info": "Info",
+  "List of nearby routes (within 1/2 km) which do not have the given stop/platform as a member.Select from the list to add the node as a member.Please note that route_ref of the node tag will also be updated accordingly." : "List of nearby routes (within 1/2 km) which do not have the given stop/platform as a member.Select from the list to add the node as a member.Please note that route_ref of the node tag will also be updated accordingly.",
   "Login": "Přihlásit",
   "Logout": "Odhlásit",
   "missing name tags": "missing name tags",
+  "Missing route_ref Tag": "Missing route_ref Tag",
+  "Missing routes": "Missing routes",
   "Monorail": "Jednokolej",
+  "Name": "Name",
   "Name/Id": "Jméno/Id",
+  "name tag errors" : "name tag errors",
+  "Nearby routes": "Nearby routes",
   "No available routes for selected feature.": "No available routes for selected feature.",
+  "No missing name suggestions available": "No missing name suggestions available",
+  "No missing ref suggestions available" : "No missing ref suggestions available",
```

Comment:
Please make sure next time that the all lines have the same format without additional spaces.